Update README.md

IrinaTerebiznik opened the pull request:
Some errors where found iside test/sample_data/README.md that prevented the test to run smoothly:
ros echo was pointing to the wrong topic
~/setup.bash was on a different path

IrinaTerebiznik committed:
Update README.md

Some errors where found iside README.md:
ros echo was pointing to the wrong topic
~/setup.bash was on a different path

IrinaTerebiznik committed:
Update README.md

Added a path specification and a source command when another tab is open

FlorGrosso commented on inorbit_republisher/test/sample_data/README.md:
````diff
@@ -21,10 +21,13 @@ topics:      /my_magnetic_field   10 msgs    : sensor_msgs/MagneticField
 ```bash
-. ~/catkin_ws/install/setup.zsh
+. ~/catkin_ws/devel/setup.zsh
+cd catkin_ws/src/inorbit_republisher/test/sample_data
````
```suggestion
cd ~/catkin_ws/src/inorbit_republisher/test/sample_data
```

FlorGrosso commented on inorbit_republisher/test/sample_data/README.md:
```diff
@@ -21,10 +21,13 @@ topics:      /my_magnetic_field   10 msgs    : sensor_msgs/MagneticField
 roslaunch sample_data.launch
 # On a different terminal windows
-$ rostopic echo /inorbit/custom_data/0
+cd catkin_ws/src/inorbit_republisher/test/sample_data
```
```suggestion
```

This step is not needed :)

FlorGrosso commented on inorbit_republisher/test/sample_data/README.md:
```diff
@@ -21,10 +21,13 @@ topics:      /my_magnetic_field   10 msgs    : sensor_msgs/MagneticField
 # On a different terminal windows
-$ rostopic echo /inorbit/custom_data/0
+cd catkin_ws/src/inorbit_republisher/test/sample_data
+source /opt/ros/noetic/setup.bash
```
```suggestion
# Optional, if you aren't sourcing your ROS workspace in .bashrc
source /opt/ros/noetic/setup.bash
```

IrinaTerebiznik committed:
Fixed path redirection on line 25 and 28 

Line 25 path redirection was missing the "~/" at the beginning.
Line 28 redirection is not necessary
